docs/news.md

```diff
@@ -6,6 +6,10 @@ toc_depth: 3
 
 ## 2025
 
+### 2025-08-26 Use of Integrated Development Environments (IDEs)
+
+Please do not use IDEs like VSCode "Remote - SSH" or Cursor Server to connect to Cheaha. These methods run all of their processes on the login node, which are automatically terminated. Instead, use "Remote - Tunnel" as described in the [VSCode Tunnel](./cheaha/open_ondemand/hpc_desktop.md#visual-studio-code-remote-tunnel) section.
```

Comment:
Same as previous. Be sure to link to the appropriate pages.

Be sure to merge in the latest changes from main, we've changed how news works.